data/human/human missions.txt

```diff
@@ -6301,3 +6301,419 @@ mission "Blind Man from Martini"
 				clear "Blind Man from Martini: label looking"
 			`	With that, he grabs his cane and his luggage, and follows a hospital attendant toward his daughter's room, leaving you in the reception.`
 
+mission "Take Me To the South A"
+	minor
+	name "Care Package for a Southern Friend"
+	source
+		government "Republic" "Syndicate"
+		not attributes "rim" "south"
+	destination
+		attributes "rim" "south"
+		not government "Pirate"
+		not system "Alniyat" "Atria" "Lesath" "Han"
+		not planet "Glaze"
+	to offer
+		not "event: war begins"
+		random < 30
+	cargo "care package" 1
+	on offer
+		conversation
+			`Among the bustle of the spaceport, one figure catches your attention. There's a woman standing alone on one side of the passageway with a bubble of space around her that she seems to be maintaining by sheer force of personality. She doesn't seem to be saying anything, at least. As you get closer to her, you notice a large package at her feet. She also notices you, and apparently something about you catches her attention, because she looks you straight in the eye and beckons.`
+			choice
+				`	(Go over to her, find out what this is about.)`
+					goto interested
+				`	(Ignore her, you don't have time for this.)`
+			`	This seems too suspicious to you, and you don't have time to be getting involved in whatever is going on here. You turn away and keep walking.`
+				decline
+			label interested
+			`	You make your way through the crowd to the edge of her personal space bubble. As soon as you do, she extends her hand. "I'm Torrey. Torrey Dupont."`
+			`	You take the hand and shake it, somewhat bemused. "<first> <last>. I'm a -"`
+			`	"An independent captain, yes. I could tell. I would like to hire you to transport a package." Torrey indicates the box beside her with a nudge of her toe.`
+			choice
+				`	"What's in the box?"`
+				`	"Wouldn't it be easier and cheaper to send it via the commercial courier services?"`
+					goto commercial
+			`	"What's in the box?"`
+			`	She glances down at it. "It's nothing too special - some local stuff from around here. Mostly foods and things that you just can't get made the same anywhere else."`
+			label commercial
+			`	"Wouldn't it be easier and cheaper to send it via the commercial courier services?"`
```

Comment:
```suggestion
			`	"Wouldn't it be easier and cheaper to send it via the commercial courier services?"`
			label commercial
```
Swap these two around so that you won't repeat yourself if you select the `commercial` label.